packages/calcite-components/src/components/loader/loader.tsx

```diff
@@ -90,9 +90,9 @@ export class Loader implements LocalizedComponent {
     return (
       <Host
         aria-label={label}
-        aria-valuemax={isDeterminate ? 100 : undefined}
-        aria-valuemin={isDeterminate ? 0 : undefined}
-        aria-valuenow={isDeterminate ? valueNow : undefined}
+        aria-valuemax={isDeterminate ? "100" : undefined}
```

Comment:
Stencil's `aria-valuemax` type is `string | number`, and at runtime, the DOM stringifies numbers automatically, so this was good.
However, the TypeScript's type for HTMLElement.ariaValuemax is `string`. Since this code is converted by the codemod to `this.el.ariaValuemax = ...`, we need to be setting strings here to avoid type error.